java/src/jmri/server/json/layoutblock/JsonLayoutBlockHttpService.java

```diff
@@ -52,7 +52,11 @@ public JsonLayoutBlockHttpService(ObjectMapper mapper) {
 
     @Override
     public JsonNode doGet(String type, String name, JsonNode data, JsonRequest request) throws JsonException {
-        return doGet(InstanceManager.getDefault(LayoutBlockManager.class).getBySystemName(name), name, type, request);
```

Comment:
It is not an internal error for a LayoutBlock to be null, so a JsonException should not indicate such. Also the doGet you replaced will either return a non-null result or throw a JsonException, so this check here for null is merely complexity for a situation that should not be possible.

Reply:
Thanks, changed to annotation.